rotkehlchen/chain/aggregator.py

```diff
@@ -889,6 +894,18 @@ def query_optimism_balances(
         """
         self.query_evm_chain_balances(chain=SupportedBlockchain.OPTIMISM)
 
+    @protect_with_lock()
+    @cache_response_timewise()
+    def query_polygon_pos_balances(
+            self,  # pylint: disable=unused-argument
+            # Kwargs here is so linters don't complain when the "magic" ignore_cache kwarg is given
+            **kwargs: Any,
+    ) -> None:
+        """
+        Queries all the polygon pos balances and populates the state.
+        Same potential exceptions as ethereum
+        """
```

Comment:
At this point I have to wonder if you are messing with me @nebolax.

This is the 3rd time I ask you to add a TODO here.

https://github.com/rotki/rotki/pull/6123#discussion_r1202793174

You can follow the links all to the first comment.
```suggestion
        POLYGON_TODO
        """
```

Reply:
Found out what's happened. I actually added POLYGON_TODO there after both the first and the second comments. But when I later ran `make format`, ruff removed it.

The way I was added the todo is by adding the following comment `# POLYGON_TODO: implement`, so the function would look like
```python
@protect_with_lock()
    @cache_response_timewise()
    def query_polygon_pos_balances(
            self,  # pylint: disable=unused-argument
            # Kwargs here is so linters don't complain when the "magic" ignore_cache kwarg is given
            **kwargs: Any,
    ) -> None:
        """
        Queries all the polygon pos balances and populates the state.
        Same potential exceptions as ethereum
        """
        # POLYGON_TODO: implement
```

When I later ran ruff, it considered the commented out todo line as commented out code (which is probably valid because a line of code like `some_variable: TypeOfTheVariable` is a valid line of code), and if it is run with `--fix` (which happens in `make format`), it automatically removes that commented out code.

-------------

I now added it in the docstring and it's fine.